README.md

```diff
@@ -176,902 +252,202 @@ Sample bands configuration file:
-  --ethfinex-api-key ETHFINEX_API_KEY
-                        API key for the Ethfinex API
-  --ethfinex-api-secret ETHFINEX_API_SECRET
-                        API secret for the Ethfinex API
-  --ethfinex-timeout ETHFINEX_TIMEOUT
-                        Timeout for accessing the Ethfinex API (in seconds,
-                        default: 9.5)
-  --pair PAIR           Token pair (sell/buy) on which the keeper will operate
-  --config CONFIG       Bands configuration file
-  --price-feed PRICE_FEED
-                        Source of price feed
-  --price-feed-expiry PRICE_FEED_EXPIRY
-                        Maximum age of the price feed (in seconds, default:
-                        120)
-  --spread-feed SPREAD_FEED
-                        Source of spread feed
-  --spread-feed-expiry SPREAD_FEED_EXPIRY
-                        Maximum age of the spread feed (in seconds, default:
-                        3600)
-  --order-history ORDER_HISTORY
-                        Endpoint to report active orders to
-  --order-history-every ORDER_HISTORY_EVERY
-                        Frequency of reporting active orders (in seconds,
-                        default: 30)
-  --refresh-frequency REFRESH_FREQUENCY
-                        Order book refresh frequency (in seconds, default: 3)
-  --debug               Enable debug output
-```
+The price feed is one of the most important determining factors of success in a Market Maker Keeper. If you have the bands set up the way you want, the price feed will help make sure your bands are set at meaningful levels relative to the inside market. If you have wide bands and your strategy is to add liquidity to handle market imbalances, then the price feed is not as important. However, as you tighten up the spreads, the price feed is a crucial component to ensure that you are going to profit in the market. 
 
+Below, we list some of the existing public feeds. You can also use web sockets if you have your own price feed that you want to use. In short, it works by each Market Maker Keeper taking in a  `--price-feed` command-line argument which then determines the price used for market-making. 
 
-## `gopax-market-maker-keeper`
+As of today, these are the possible values of this argument ( existing public feeds)  that we list:
 
-This keeper supports market-making on the [GoPax](https://www.gopax.co.kr/) exchange.
+- `fixed:<amount>` - uses a fixed price (`fixed:200`in this example). See below for a more in-depth example. Note that when you are on mainnet, you typically won't use a fixed amount but it is an ideal example for this walkthrough as there aren't price feeds for Kovan.
+- `eth_dai` - uses the price from the GDAX WebSocket ETH/USD price feed.
+- `eth_dai-setzer` - uses the average of Kraken and Gemini ETH/USD prices.
+- `eth_dai-tub` - uses the price feed from `Tub` (only works for keepers being able to access an Ethereum node).
+- `eth_dai-pair` - uses the price from the GDAX WebSocket ETH/DAI price feed.
```

Comment:
I think it may be a good idea anytime GDAX is listed in this doc we have in parentheses -> `(Coinbase)` since CB bought GDAX and newcomers to the space may not know what GDAX is